app/models/update_existing_draft_edition.rb

```diff
@@ -79,4 +95,20 @@ def attributes
   def edition_attributes_from_payload
     payload.slice(*Edition::TOP_LEVEL_FIELDS)
```

Comment:
`ATTRIBUTES_TO_BE_ADDED` is now a hash called `DATE_FIELDS` so I won't be able to combine it with `TOP_LEVEL_FIELDS` anymore.